src/OpenTelemetry.Exporter.Geneva/Internal/Transports/UnixUserEventsDataTransport.cs

```diff
@@ -0,0 +1,51 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#if NET
+
+using Microsoft.LinuxTracepoints.Provider;
+
+namespace OpenTelemetry.Exporter.Geneva.Transports;
+
+internal sealed class UnixUserEventsDataTransport : IDisposable
+{
+    public const string EventHeaderDynamicProviderName = "MicrosoftOpenTelemetryLogs";
+    public const System.Diagnostics.Tracing.EventLevel LogsTracepointEventLevel = System.Diagnostics.Tracing.EventLevel.Informational;  // TODO: find the correct event level
```

Comment:
Can we shorten this `System.Diagnostics.Tracing.EventLevel` to `EventLevel`?

Reply:
Updated.